client/src/pages/CalendarPage/CalendarPage.js

```diff
@@ -98,6 +102,8 @@ const CalendarPage = () => {
   };
 
   useEffect(() => {
+    setToggleError(false);
+    setLoadingError(false);
```

Comment:
We can remove this since it's already in the `getTasks` function

Reply:
Removed.